src/parent-steps/automation/execute-automation-step.ts

```diff
@@ -111,9 +111,9 @@ export class ExecuteAutomationStep extends AutomationStep {
 
   listOutputs(): Output[] {
     return [{
-      outputType: DataTypeEnum.STRING_LIST,
+      outputType: DataTypeEnum.STRING_MAP,
```

Comment:
The action actually outputs a list:
https://docs.aws.amazon.com/systems-manager/latest/userguide/automation-action-executeAutomation.html

```
Output

Output

    The output generated by the secondary automation. You can reference the output by using the following format: Secondary_Automation_Step_Name.Output

    Type: StringList
```

The list is ordered by the outputs that are requested for execution.

Reply:
Confirmed the documentation is correct.  Reverted the change.